easyCore/__init__.py

```diff
@@ -5,11 +5,10 @@
 __author__ = "github.com/wardsimon"
 __version__ = "0.3.1"
 
-import numpy as np
 
-from easyCore.Objects.Borg import Borg
 import pint
-from .REDIRECT import _REDIRECT
+
+from easyCore.Objects.Borg import Borg
```

Comment:
Same as `np` - libs/apps import `borg` directly from easyCore with

`from easyCore import Borg`

which is more intuitive than importing Borg from some internal easyCore path, which might change after refactoring.

Reply:
oh! never mind. I'm referring to me being blind and not seeing this was just a line shift...